### PR TITLE
[bitnami/jenkins] Stabilize Jenkins by adding a replicaCount

### PR DESCRIPTION
**Pull request by elatella**
### Description of the change

Add replicaCount to declare replicaCout in Values 

### Benefits

By declaring the replicaCount, it removes the possibility to have the replicas scaled up or down my external applications (ArgoCD, ...)

### Applicable issues

- fixes #

### Checklist

- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/). This is *not necessary* when the changes only affect README.md files.
- [ x ] Variables are documented in the values.yaml and added to the `README.md` using [readme-generator-for-helm](https://github.com/bitnami/readme-generator-for-helm)
- [ x ] Title of the pull request follows this pattern [bitnami/<name_of_the_chart>] Descriptive title
- [ x ] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/main/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

**Comment by elatella** [comment]
@chrira please check

File: bitnami/jenkins/values.yaml [hunk 1]
```diff
@@ -40,6 +40,9 @@ nameOverride: ""
 fullnameOverride: ""
 ## @param commonLabels Labels to add to all deployed objects
 ##
+replicas: {}
```
Comment (chrira) [inline bitnami/jenkins/values.yaml:43]: Comment has to be on top of the key.
Define it like here: https://github.com/bitnami/charts/blob/main/bitnami/apache/values.yaml#L108

File: bitnami/jenkins/templates/deployment.yaml [hunk 2]
```diff
@@ -16,6 +16,9 @@ spec:
   {{- $podLabels := include "common.tplvalues.merge" ( dict "values" ( list .Values.podLabels .Values.commonLabels ) "context" . ) }}
   selector:
     matchLabels: {{- include "common.labels.matchLabels" ( dict "customLabels" $podLabels "context" $ ) | nindent 6 }}
+  {{- if .Values.replicaCount }}
```
Comment (chrira) [inline bitnami/jenkins/templates/deployment.yaml:19]: Change it to this solution: https://github.com/bitnami/charts/blob/main/bitnami/apache/templates/deployment.yaml#L19

File: bitnami/jenkins/README.md [hunk 3]
```diff
@@ -196,6 +196,7 @@ s
 | `kubeVersion`            | Override Kubernetes version                                                             | `""`            |
 | `nameOverride`           | String to partially override common.names.fullname                                      | `""`            |
 | `fullnameOverride`       | String to fully override common.names.fullname                                          | `""`            |
+| `replicaCount`           | Number of containers                                                                    | `1`             |
```
Comment (chrira) [inline bitnami/jenkins/README.md:199]: Do not manually change the table.
Generate it like described: https://github.com/bitnami/charts/blob/main/CONTRIBUTING.md#documentation-requirements

**Review by chrira — CHANGES_REQUESTED** [review]
Thanks for the implementation.
Please forget my initial idea with the complex logic for having it defined or not.
When needed the autoscaling feature could be added.
In this PR I would just add the replicaCount with a fix number of 1.

**Comment by javsalgar** [comment]
Hi!

Thank you so much for the PR! Could you bump the chart version?

File: bitnami/jenkins/templates/deployment.yaml (hunk 2 above)
Comment (elatella) [reply bitnami/jenkins/templates/deployment.yaml]: Fixed

**Comment by carrodher** [comment]
There are some failing actions:
```
Error: Detected changes in charts without version bump in Chart.yaml. Charts changed: 1. Version bumps detected: 0
```

You need to bump the chart version to properly release a new Helm chart with those changes

**Review by chrira — APPROVED** [review]
LGTM and is aligned with other bitnami charts.
Thank you!

**Review by dgomezleon — APPROVED** [review]
LGTM

**Review by dgomezleon — APPROVED** [review]

**Pull request merged**